pkgs/top-level/python-packages.nix

```diff
@@ -12137,13 +12154,13 @@ in modules // {
     };
 
     # most of these are simply to allow the test suite to do its job
-    buildInputs = with self; [ mock unittest2 nose redis qpid-python pymongo sqlalchemy pyyaml msgpack boto ];
+    buildInputs = with self; optionals isPy27 [ mock unittest2 nose redis qpid-python pymongo sqlalchemy pyyaml msgpack boto ];
```

Comment:
This is because it can only be tested with python 2.7. Some dependencies required for tests (`qpid-python`) are not available for python3, but the final release is said to be fine on python 3 (and required by other projects such as `celery`). I would prefer to test each version, but I do not see how…